src/chai-form.ts

```diff
@@ -346,22 +367,24 @@ export class ChaiForm extends LitElement {
       <slot name="before"></slot>
       <form id="chai-quote-form">
         <slot>
-          <chai-name></chai-name>
-          <chai-phone></chai-phone>
-          <chai-email></chai-email>
-          <chai-address></chai-address>
+          <chai-name id="chai-name"></chai-name>
```

Comment:
It should not be necessary to set `id` attributes on these default slot elements. First, because a single webpage can have multiple `<chai-form>`s on it. Second, because this doesn't include `<chai-date>`.